controller/src/main/java/io/pravega/controller/server/ControllerService.java

```diff
@@ -180,7 +180,7 @@
                                         context,
                                         executor)
                                         .thenApply(seg -> ModelHelper.createSegmentRange(segment.getStreamInfo().getScope(),
-                                                segment.getStreamInfo().getStream(), seg.getSegmentId(),
+                                                segment.getStreamInfo().getStream(), seg.segmentId(),
```

Comment:
this is because in GetScalingMetadata api for rest, we pass this segment object which is converted to json and sent over the wire. it takes all fields with getters and tries to put them in the serialized object. 
so `get` verb in this class will interfere with that logic. 

as part of #2680 we want to only pass segment number and epoch as response to the getScalingMetadata api. those can be correlated with the naming in the metrics which explicitly puts segmentNumber and epoch.